### ISSUE TITLE
WCAG 4.1.2: ARIA hidden element must not contain focusable elements (.input-group-text) Tags: Accessibility, WCAG 4.1.2, aria-hidden-focus

### ISSUE THREAD

**Issue by brianteeman** [joomla/backend-template#98]
This commit has created a focusable element (button) on the admin login page which is hidden https://github.com/joomla/backend-template/commit/dac783ee81595267548c83131383f3b9e76a012f

Issue: ARIA hidden element must not contain focusable elements (aria-hidden-focus: https://dequeuniversity.com/rules/axe/3.2/aria-hidden-focus?application=msftAI)

Target application title: backend design - Administrator Login
Target application url: http://localhost/backend-template-release/backend-template-release/administrator/index.php

Element path: .input-group-text

Snippet: `<button type="button" class="input-group-text icon-eye input-password-toggle" aria-hidden="true"></button>`

**Comment by coolcat-creations** [comment]
thank you

**Comment by zwiastunsw** [comment]
1. The button does not have an accessible name.
2. The button cannot be hidden for screen readers. 
3. The "Show" command (name of the button) is located in the wrong place. It should be placed inside the `button` tag.
4. The Show button should be in the order of tabulation in front of the password field. Placing it behind the Password field is of little use to screen reader users. If they want to hear if they are typing the password correctly, they must use the Show command before they start typing it. Otherwise, the reader will not read the password entered to them, and during typing it will read "star star star star ...." instead of consecutive letters.  
  
The code should look like this:

```
<span class="input-group-append ml-2">
   <button type="button" class="input-group-text icon-eye input-password-toggle" >
         <span class="sr-only"><?php echo Text::_('JSHOW'); ?></span>
   </button>
</span>
```

**Comment by coolcat-creations** [comment]
Thank you, committed your code in 07e1188322207f1c6f0fc0dd32d2e678a954d130 [07e1188]
